src/screens/SendSheet.js

```diff
@@ -347,6 +304,87 @@ export default function SendSheet(props) {
     [selected, sendUpdateAssetAmount, updateMaxInputBalance]
   );
 
+  // Update all fields passed via params if needed
+  useEffect(() => {
```

Comment:
do you usually put `useEffect`s in the middle of `useState`s?

Reply:
only when I code at 3AM lol. Just fixed it 😉

Reply:
fast af